bin/conveyor

```diff
@@ -1,7 +1,10 @@
 #!/usr/bin/env ruby
-$LOAD_PATH.unshift(File.expand_path(File.join(File.dirname(__FILE__), "..", "lib")))
+$LOAD_PATH.unshift(File.join(File.dirname(File.realpath(__FILE__)), "..", "lib"))
```

Comment:
Wouldn't you want the expand path still around the `File.join` to deal with the `../lib` that gets appended?